fdbserver/Ratekeeper.actor.cpp

```diff
@@ -1050,6 +1054,18 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 			} else {
 				ssLimitReason = limitReason_t::storage_server_min_free_space_ratio;
 			}
+			if (printRateKeepLimitReasonDetails) {
+				TraceEvent("RatekeeperLimitReasonDetails")
+				    .detail("Reason", ssLimitReason)
+				    .detail("SSID", ss.id)
+				    .detail("ServerKnobsMinAvailableSpace", SERVER_KNOBS->MIN_AVAILABLE_SPACE)
```

Comment:
We don't need to print KNOBS value. These knobs value can be found in code and changed value can be found in ProgramStart message.

Pls remove the Knobs in logging